agent/wsclient/client.go

```diff
@@ -60,6 +63,8 @@ const (
 
 	// Default NO_PROXY env var IP addresses
 	defaultNoProxyIP = "169.254.169.254,169.254.170.2"
+
+	errClosed = "use of closed network connection"
```

Comment:
Can we have the error name more descriptive like `errClosedConnection` or similar?

Reply:
`errClosed` is pretty descriptive, no? I can change it you have a *strong* preference